octobot_trading/api/orders.py

```diff
@@ -36,3 +40,23 @@ def get_open_orders(exchange_manager, symbol: str):
                                                                                                  current_price=current_price,
                                                                                                  quantity=quantity,
                                                                                                  price=price))
+
+
+def get_open_orders(exchange_manager):
+    return exchange_manager.exchange_personal_data.orders_manager.get_open_orders()
+
+
+async def cancel_all_open_orders(exchange_manager):
+    return await exchange_manager.trader.cancel_all_open_orders()
+
+
+async def cancel_all_open_orders_with_currency(exchange_manager, currency):
+    return await exchange_manager.trader.cancel_all_open_orders_with_currency(currency)
+
+
+async def cancel_order_from_description(exchange_manager, order_description):
```

Comment:
I didn't like the implementation of OctoBot < 0.3.7 and removed it.

Now each order (simulated or real) has an order_id.

So this method should be cancel_order_from_id

Reply:
Good !